funcx/endpoint/endpoint.py

```diff
@@ -147,6 +147,33 @@ def register_with_hub(address, redis_host='funcx-redis.wtgh6h.0001.use1.cache.am
 
     return r.json()
 
+def configure_endpoint(args, global_config=None):
```

Comment:
Maybe for discussion or future work, but can we adjust the contents of the config from here? (e.g., funcx-endpoint configure my_endpoint --max_workers=12, ...) My only rationale for this is from being a developer person who doesn't want to change the endpoint config each time I spin up a new endpoint (and I currently keep accidentally committing my changes to the default config) :)

Reply:
I know this is going to be a nuisance for us since we'll be doing a lot of restarts. For now let's start with a template, and make an issue on finding a way to make dev easier. Maybe pass the configure command a config file path as an arg ?

Reply:
Fixed.